source/usage-examples/findOne.txt

```diff
@@ -3,3 +3,75 @@ Find a Document
 ===============
 
 .. default-domain:: mongodb
+
+You can retrieve a single document from your collection with the
+``*Collection.findOne()`` method. 
```

Comment:
Not sure you meant to do this, but let's not use "*" to prefix a method name. Also, as noted in a prior comment, the "f" needs to be capitalized. That is the golang convention for calling (public) methods in a package. Could you make sure you address all instances as requested inthe prior comment: https://github.com/mongodb/docs-golang/pull/2#discussion_r608760490

Reply:
Thanks for the note. I apologize for misunderstanding your previous message. I incorrectly read it as "oh no my exported type is not capitalized!" rather than "oh no my exported type and exported method are not capitalized!" I will try and not make this mistake in the future.

I have removed the pointer prefix for the `FindOne()` method. Is it safe to say that going forward when discussing go methods, we will not distinguish between pointer receivers and struct receivers when discussing go methods?

When discussing the pointer receiver methods such as [`FindOne`](https://pkg.go.dev/github.com/mongodb/mongo-go-driver/mongo#Collection.FindOne), we should refer to them as `Collections.FindOne()` rather than `*Collections.FindOne()`?

For function/method signatures, should we distinguish between recieving a pointer type and a non-pointer type? So in the case of a method receiving a pointer to ``SingleResult`` variable, should we say the method takes a. ``*SingleResult`` or a ``SingleResult``?

Just trying to nail down how we plan to talk about pointers in the go docs.

Fix: